cmd/spruce/main_test.go

```diff
@@ -1980,6 +1981,73 @@ meta:
 
 `)
 		})
+
+		Convey("Issue #267 Implicit/default merge behaves different than explicit merge", func() {
```

Comment:
Can you change the name of this to reflect how the behavior should be as a result of fixing the bug, rather than the errant behavior / bug title?

Reply:
Yes, that makes much more sense that way. I will change that.